Yoast/wordpress-seo
# Replaced 'Basic SEO course' in the dashboard sidebar with `All-Around SEO training'.

## Summary

This PR can be summarized in the following changelog entry:

* Replaces the reference to the Basic SEO training in the dashboard sidebar with a reference to the All-Around SEO training.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* In the dashboard, check that the suggested courses in the sidebar look as follows:
<img width="1000" alt="53345508-600be700-3915-11e9-90af-d36bdd1670a6" src="https://user-images.githubusercontent.com/20280513/53403375-d19a7280-39b3-11e9-8393-af2eefcacee1.png">

* Check out this branch.
* The suggested `Basic SEO course` in the sidebar should now be replaced by the `All-Around SEO training`.
* The text `Learn practical SEO skills to rank higher in Google.` should not have changed.
* The icon (of the kite) should have been with a picture of a pocket knife.
* For the other suggested courses as well, the word `course` should have changed into `training`.
* All in all, this is how the changes should look:
<img width="1000" alt="screen shot 2019-02-26 at 10 41 09" src="https://user-images.githubusercontent.com/20280513/53403605-42da2580-39b4-11e9-80c7-c4c52a2e3cb1.png">

* When you click on the `All-Around SEO training`, you currently get a `Page not found` error. This link should start working fine when 10.0 is released.

## UI changes

## Quality assurance


Fixes #12313
